buildsystem/scripts/EmbedWinDependencies.cmake

```diff
@@ -11,6 +11,21 @@ if(NOT dumpbin)
 	return()
 endif()
 
+# Paths for applocal and powershell
+set(APPLOCAL_SCRIPT "${vcpkg_dir}/../../scripts/buildsystems/msbuild/applocal.ps1")
+set(POWERSHELL_COMMAND "C:/Windows/System32/WindowsPowerShell/v1.0/powershell.exe")
+
+# Check if applocal exists
+if(NOT EXISTS ${APPLOCAL_SCRIPT})
+	unset(APPLOCAL_SCRIPT)
+endif()
+
+# Check if powershell exists
+if(NOT EXISTS ${POWERSHELL_COMMAND})
```

Comment:
Micro-optimization: The two checks can be moved in the `if` below, instead of unsetting the variable here and checking for `DEFINED` later.